inference-engine/tests/functional/plugin/shared/include/behavior/infer_request.hpp

```diff
@@ -623,4 +624,39 @@ TEST_P(InferRequestTests, returnDeviceBusyOnGetPerformanceCountAfterAsyncInfer)
         std::cout << "Exception" << e.what() << std::endl;
     }
 }
-}  // namespace BehaviorTestsDefinitions
\ No newline at end of file
+
+class InferRequestTestsResultNotReady : public InferRequestTests {
+};
+
+TEST_P(InferRequestTestsResultNotReady, ReturnResultNotReadyFromWaitInAsyncModeForTooSmallTimeout) {
+    // Skip test according to plugin specific disabledTestPatterns() (if any)
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    // Create CNNNetwork from ngrpah::Function
+    // return function which computes around 20ms on GNA SW
+    function = LayerTestsDefinitions::Basic_LSTM_S::GetNetwork(3000, 380);
+    InferenceEngine::CNNNetwork cnnNet(function);
+    // Load CNNNetwork to target plugins
+    auto execNet = ie->LoadNetwork(cnnNet, targetDevice, configuration);
```

Comment:
For what execution mode do you load the network? I see that the configuration is empty and the default values are GNA_SW_EXACT with scale factor equals 1.0f.

Reply:
IMO it does not matter for this test, I don't want to compare scores etc. Other tests in this set (infer_request.hpp) looks similar.